### DISCUSSION TITLE
How can I make breadcrumb's link openable in new tab or middle mouse clickable ?

### DISCUSSION THREAD

**Discussion by borisBelloc** [udayvunnam/xng-breadcrumb#80] [Q&A] [answered]
Hello,

Is it possible to make the breadcrumb's link middle or right mouse clickable to open it in new tab ?

Currently, if you try to click a link with the middle mouse button, nothing happens.

![image](https://user-images.githubusercontent.com/37147299/113578545-3e68a800-9623-11eb-8ef7-1ab8d41f098e.png)

**I'm not sure** but I think I'm looking to make the link `[href]` rather than using `[routerlink]`

any help ?
thank you

**Comment by udayvunnam** [comment]
Hi @borisBelloc do you want this behaviour for all links in breadcrumbs or do you want to apply this behaviour conditionally on certain links

**Reply by borisBelloc** [reply]
hi,
I would want it for all links.

**Comment by udayvunnam** [comment]
I will add this feature shortly. A PR with the change is welcome too :)

**Reply by borisBelloc** [reply]
thanks a lot; 

> a PR with the change is welcome too :)

I tried to see what's needed but I'm not good enough to do it 😲

is adding `href="#"`  ok for you ? 😶
![image](https://user-images.githubusercontent.com/37147299/113603198-ee4e0d80-9643-11eb-9141-3ae950c6be14.png)

**Comment by udayvunnam** [comment, marked as answer]
@borisBelloc This feature is added with version `6.7.0` 🧨

- You can now `cmd + click` or `middle mouse click` to open the link in the new tab 
- If you wish to open all breadcrumb links in a new tab on click you can do so by https://github.com/udayvunnam/xng-breadcrumb/blob/main/docs/advanced-patterns.md#open-breadcrumb-link-in-new-tab

**Reply by borisBelloc** [reply]
thank you
